src/child/index.js

```diff
@@ -19,19 +21,26 @@ require('script!../../node_modules/d3fc/dist/d3fc.bundle.min.js');
 require('script!../../node_modules/BitFlux/dist/bitflux.js');
 /* eslint-enable import/no-unresolved */
 
-fin.desktop.main(() => {
-    fin.desktop.Window.getCurrent().contentWindow.addEventListener('beforeunload', () => {
-        fin.desktop.Window.getCurrent().contentWindow.opener.store.dispatch(close());
-        unmountComponentAtNode(document.getElementById('app'));
+currentWindowService.ready(() => {
+    const currentWindow = currentWindowService.getCurrentWindow();
+    const store = currentWindow.contentWindow.opener.store;
+    const rootElement = document.getElementById('app');
+
+    const windowStateService = new WindowStateService(currentWindow, store);
+    windowStateService.start();
+
+    currentWindow.contentWindow.addEventListener('beforeunload', () => {
+        store.dispatch(close());
```

Comment:
I think we need to unmount before dispatching close and stopping the service.
I've fixed that in another PR, but just in case this gets merged first